omero_figure/templates/figure/index.html

```diff
@@ -605,6 +605,11 @@ <h4>Dynamic properties</h4>
                             <td>Dataset ID: [dataset.id]</td>
                             <td>Dataset ID: 1654</td>
                         </tr>
+                        <tr>
+                            <th>Zoom</th>
+                            <td>[zoom] %</td>
```

Comment:
The ` %` is generated by the `[zoom]` itself, so you have `[zoom] %` you actually get `[zoom] % %`.
So I think you can just remove the ` %` from this line.

Reply:
It takes me moment but now I get it. Yes, we can remove the `` %`` from the tips.